Programming_Guides/Opencl-programming-guide.rst

```diff
@@ -2967,7 +2967,7 @@ There are special directives for the OpenCL compiler to enable or disable availa
  #pragma OPENCL EXTENSION all: <behavior>
 
 
-The <extension_name> is described in Section A.1, “Extension Name
+The <extension_name> is described in Section A.1, "Extension Name
 Convention.”. The second form allows to address all extensions at once. The <behavior> token can be either:
```

Comment:
Looks like the starting quote was considered part of "code", while the closing quote wasn't.